src/main/java/no/fintlabs/consumer/model/leverandor/LeverandorConfig.java

```diff
@@ -12,16 +12,6 @@ public LeverandorConfig(ConsumerProps consumerProps) {
         super(consumerProps);
     }
 
-    @Override
-    protected String domainName() {
```

Comment:
Når disse er tatt vekk herfra, må de ikke legges i kustomize-base i stede for? (så ingen endring påd et)